css/cart.css

```diff
@@ -0,0 +1,199 @@
+.cart {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 2rem 0rem;
+  border-bottom: 1px solid var(--header-backgroundColor);
+}
+.cart p {
+  font-size: 2rem;
+  font-weight: bold;
+}
+.checkout-details {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin-top: 2rem;
+}
+.final-checkout {
+  display: flex;
+  justify-content: space-between;
+}
+.cart-products {
+  display: flex;
+  flex-direction: column;
+
+  align-items: center;
+}
+.checkout-list {
+  display: flex;
+  justify-content: space-between;
+  width: 70%;
+}
+.product-img {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 60%;
+  padding: 0rem 1rem 0rem 0rem;
```

Comment:
suggestion : you can use padding right here.

Reply:
Done